maxim/logger/fireworks/utils.py

```
@@ -0,0 +1,347 @@
+                represents the complete response text.
+            usage_data (Any): Usage information from the final chunk
+                containing token counts and other usage metrics.
+                
+        Returns:
+            Response: A structured response object that can be processed by
+                the parse_completion method for consistent logging.
+        """
+        return Response(content, usage_data)
+    
+    @staticmethod
+    def parse_completion(completion: ChatCompletion) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)